ecms_base/features/custom/ecms_hotel/config/install/core.entity_form_display.node.hotel.default.yml

```diff
@@ -0,0 +1,162 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.node.hotel.field_hotel_address
+    - field.field.node.hotel.field_hotel_body
+    - field.field.node.hotel.field_hotel_booking_notes
+    - field.field.node.hotel.field_hotel_booking_url
+    - field.field.node.hotel.field_hotel_cover_image
+    - field.field.node.hotel.field_hotel_main_image
+    - field.field.node.hotel.field_hotel_more_info_url
+    - field.field.node.hotel.field_hotel_phone
+    - field.field.node.hotel.field_hotel_rate
+    - field.field.node.hotel.field_hotel_rate_amount
+    - node.type.hotel
+    - workflows.workflow.editorial
```

Comment:
Should the workflow be a dependency? I believe the workflows get applied programmatically and not through configuration.

Reply:
Yep - saw that elsewhere. Going to remove.

Reply:
I think I got rid of all of these, plus some content_moderation references